kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -46,7 +46,7 @@ pub struct LookupPatterns {
     pub xor: bool,
     pub lookup: bool,
     pub range_check: bool,
-    pub foreign_field_mul: bool,
+    pub ffmul: bool,
```

Comment:
This is now renamed in the opposite direction. Revert.

Reply:
Oh man, now I refactored even further... Fixing it